serving/docs/lmi/user_guides/chat_input_output_schema.md

```diff
@@ -0,0 +1,259 @@
+# Chat Completions API Schema
+
+This document describes the API schema for the chat completions endpoints (`v1/chat/completions`) when using the built-in inference handlers in LMI containers.
```

Comment:
Can we state that this is an experimental feature, and the complete spec has not been implemented? We can also link to the openAi spec.

Reply:
Changed.